rest-api/src/main/java/com/cloudbees/workflow/flownode/FlowNodeUtil.java

```diff
@@ -73,9 +65,7 @@ private FlowNodeUtil() {
     }
 
     public abstract static class CacheExtensionPoint implements ExtensionPoint {
-        public abstract Cache<String,List<FlowNode>> getExecutionCache();
```

Comment:
FWIW my original recommendation was for there to _not_ be an `ExtensionPoint`, since there was never any expectation of multiple implementations—just a singleton `@Extension`.